misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/variable/types/BTable.java

```diff
@@ -46,11 +46,17 @@
 
     private int tableSize = -1;
     private ArrayReference tableKeys = null;
+    private ArrayList<Value> valueList = new ArrayList<>();
 
     private static final String FIELD_CONSTRAINT = "constraint";
     private static final String METHOD_SIZE = "size";
     private static final String METHOD_GETKEYS = "getKeys";
     private static final String METHOD_GET = "get";
```

Comment:
Can we remove these constants, if not being used anymore?

Reply:
Fixed